it.js

```diff
@@ -1,863 +1,863 @@
+server.config.nameTeam1 Nome squadra 1
+server.config.nameTeam2 Nome squadra 2
+server.config.nameTeam3 Nome squadra 3
+server.config.nameTeam4 Nome squadra 4
+server.config.nameTeam5 Nome squadra 5
+server.config.t1Dmg Danni squadra 1
+server.config.t2Dmg Danni squadra 2
+server.config.t3Dmg Danni squadra 3
+server.config.t4Dmg Danni squadra 4
+server.config.t5Dmg Danni squadra 5
+server.config.selTeam Scelta squadra
+server.config.allowSpect Spettatori
+server.config.killRewards Premi uccisioni
+server.config.headshotOnly Solo headshot
+server.config.noSecondary Disabilita secondarie
+server.config.noStreaks Disabilita serie di uccisioni
+server.config.canSlide Scivolare
+server.config.wallJP  Salto dal muro
+server.config.autoJump Salto automatico
+server.config.bDrop Cadenza proiettili
+server.config.thirdPerson 3a persona
+server.config.requireCaptcha Richiedi Captcha
+server.config.nameTags Nascondi Nametags
+server.config.kCams Camera uccisione
+server.config.noReload Niente ricarica
+server.config.private Privata
+server.config.krRewards Premi KR
+server.config.forceDefault Forza impostazioni predefinite
+server.config.warmupTime Tempo riscaldamento
+server.config.gamRounds Rounds
+server.config.objtvTime Tempo obiettivo
+server.config.forceSpawn Tempo rinascita automatica
+server.config.intermTmr Intervallo (s)
+server.config.scoreLimit Limite punteggio
+server.config.keepTScore Tieni punti squadra
+server.config.forceC Forza la classe
+server.config.logTim Tempo (Gara, Parkour)
+server.config.hitBoxPad Dimensione Hitbox
+server.config.hpRegen Rigenerazione vita
+server.config.disableB Disabilita bordi
+server.config.throwMel Armi da mischia lanciabili
+server.config.noCosm Disabilita cosmetici
+server.config.tstCmp Modalità test
+server.config.limitClasses Limita classi
+server.config.tmSize Dimensione squadra
+server.config.fallDmg Danno da caduta
+server.config.fallDmgThr Soglia del danno da caduta 
+server.config.noDraws Nessun pareggio
+server.config.bstOfR Al meglio di
+server.config.chrgWeps Chargable Weapons
+social.login Login/Registrati
+social.login.failed C'è stato un errore nel login!
+social.login.buy Effettua il login per comprare
+social.login.sell Effettua il login per vendere
+social.login.gift Effettua il login per regalare
+social.login.proccessing Login in corso...
+social.since Dal
+social.user.maps Mappe degli utenti
+social.user.mods Mod degli utenti
+social.maps.none Nessuna mappa trovata...
+social.mods.none Nessuna mod trovata...
+social.market.price.low Prezzo più basso
+social.market.price.list Elenca prezzi dal
+social.market.searching Cercando
+social.market.listed Aggiunto {0} fa
+social.market.list Metti in vendita
+social.market.unlist Rimuovi
+social.market.unlisting Rimuovendo oggetto...
+social.market.posting Mettendo in vendita...
+social.market.req richiesto lvl {0}
+social.market.cant Non puoi permettertelo
+social.market.purchase Acquista
+social.market.fee Gratis!

```

Comment:
Personally I would drop the exclamation mark, but I would say the translation works.